client/app/hosting/user-logs/USER_LOGS.html

```diff
@@ -44,78 +44,29 @@ <h2 data-i18n-static="hosting_tab_USER_LOGS"></h2>
-                                        <button class="btn btn-icon" type="button"
-                                                data-linkedpopover="hosting/popover/user_logs.html"
-                                                data-linkedpopover-placement="left"
-                                                data-linkedpopover-single="true"
-                                                data-linkedpopover-remote="true"
-                                                data-linkedpopover-title="{{i18n.common_actions}}">
-                                            <span class="oui-icon oui-icon-gear_line oui-icon_small" aria-hidden="true"></span>
-                                        </button>
-                                    </td>
-                                </tr>
-                            </tbody>
-                        </table>
-                    </div>
-                </div>
-
-                <div class="clearfix"
-                     data-ng-if="ctrl.userLogs.ids">
-                    <div data-pagination-front
-                         data-items="ctrl.userLogs.ids"
-                         data-paginated-items="ctrl.userLogs.details"
-                         data-current-page="currentPage"
-                         data-nb-pages="nbPages"
-                         data-items-per-page="itemsPerPage"
-                         data-transform-item="ctrl.transformItem(item)"
-                         data-on-transform-item-done="ctrl.onTransformItemDone(items)"
-                         data-page-placeholder="{{tr('pagination_page', [currentPage, nbPages])}}"
-                         data-item-per-page-placeholder="{{::tr('pagination_display')}}"
-                         data-on-page-change="ctrl.loaders.pager = true"
-                         data-ng-show="!ctrl.loaders.userLogs && ctrl.userLogs.details.length > 0">
-                    </div>
-                </div>
+                <oui-datagrid data-rows="ctrl.userLogs.ids"
+                              data-row-loader="ctrl.transformItem($row)">
+                    <oui-column data-title="tr('hosting_tab_USER_LOGS_table_header_login')" data-property="login">
+                        <span data-ng-bind="$row.login"></span>
+                    </oui-column>
+                    <oui-column data-title="tr('hosting_tab_USER_LOGS_table_header_description')" data-property="description">
+                        <span data-ng-bind="$row.description | date:'short'"></span>
+                    </oui-column>
+                    <oui-column data-title="tr('hosting_tab_USER_LOGS_table_header_creationDate')" data-property="creationDate">
+                        <span data-ng-bind="$row.creationDate | date: 'medium'"></span>
+                    </oui-column>
+                    <oui-action-menu align="end" compact>
+                        <oui-action-menu-item text="{{::tr('hosting_tab_USER_LOGS_table_popover_modify')}}"
```

Comment:
Weren't you supposed to add `data-` to all custom attributes ?

Reply:
Yes, will fix

Reply:
Neat :)